sys/dev/axgbe/if_axgbe_pci.c

```diff
@@ -56,6 +56,14 @@
 #include "ifdi_if.h"
 #include "opt_inet.h"
 #include "opt_inet6.h"
+#include "opt_rss.h"
+
+#ifdef RSS
+
```

Comment:
We don't normally have blank lines like this after the ifdef line, before the extra includes (or before the final endif.